packages/teams-js/src/public/constants.ts

```diff
@@ -49,15 +61,23 @@ export enum HostName {
   teamsModern = 'TeamsModern',
 }
 
-// Ensure these declarations stay in sync with the framework.
+/** FrameContexts provides information about the context in which the app is running within the host. */
 export enum FrameContexts {
+  /** Allows developers to access and modify the configuration settings of a host. */
```

Comment:
>/** Allows developers to access and modify the configuration settings of a host. */ [](http://example.com/codeflow?start=2&length=83)

I'm not sure this description is accurate or helpful. I don't know what the "settings" frame context is for and I'm not sure I totally understand after reading this addition.

If this is the same thing as a configuration page, consider maybe linking to here?
[https://learn.microsoft.com/microsoftteams/platform/tabs/how-to/create-tab-pages/configuration-page?tabs=teamsjs-v2](https://learn.microsoft.com/microsoftteams/platform/tabs/how-to/create-tab-pages/configuration-page?tabs=teamsjs-v2)